### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by GrayTurtle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,11 +115,29 @@ Classes that are shared by the clients and servers.
...
+![alt text](https://raw.githubusercontent.com/GrayTurtle/codeu_project_2017/develop/images/Chat.png)
+- Updated the client-side and server-side controllers to interface with new GUI and database
+- Integrated a DERBY local database to save chat information
+- The chat is saved, and once users sign in it is the same as they had left it
+
 
 ## PLANNED CHANGES
+
+- Implement leaving a conversations (we currently just have a button for it)
+- Make the client update faster by changing out the timer for something else
```
Comment (MalikG15) [inline README.md]: What do you mean by changing out the timer?

**Comment by MalikG15** [comment]
You did a great job with this John!

File: README.md (hunk 1 above)
Comment (GrayTurtle) [reply README.md]: Like instead having it update every second or something, update whenever it needs to? I remember you talking about something like that

**Review by KevinWorkman — COMMENTED** [review]
Nice work. The only thing I would change is I would get rid of the boiler plate code (basically everything before your new "our changes and new features" section).

I'd also include a section right at the top saying that this requires Oracle Java. Followed by instructions on how to run it- did you get rid of the team secret?

Keep in mind that reviewers are going to see a bunch of these readme files, and I would guess that most teams are going to be pretty bad at updating them. So make it obvious that you updated yours by moving the new stuff to the top instead of "buring the lede" if that makes sense.

You might also think about copying some of this stuff into the design doc. It's an easy win, so why not.

**Comment by KevinWorkman** [comment]
Actually I just realized you already did a lot of what I recommended above.

So the only thing I'd recommend is just making it move obvious. For example maybe change the big title at the top to "Team 27 CodeU Chat Program" or something (did you give it a name?) and you can get rid of the disclaimer and the environment that came from the original repo. Maybe make sure the Oracle Java disclaimer is more obvious: we don't recommend it, it's absolutely required, right?

But anyway at this point I'm probably just nit-picking. :p

File: README.md (hunk 1 above)
Comment (MalikG15) [reply README.md]: It does update when it needs to already though. There is no timer.

**Pull request merged**